Modules/BenchPress.Azure/Public/Confirm-ApiManagementPolicy.ps1

```diff
@@ -0,0 +1,62 @@
+# INLINE_SKIP
+using module ./../Classes/ConfirmResult.psm1
+
+. $PSScriptRoot/../Private/Connect-Account.ps1
+# end INLINE_SKIP
+
+function Confirm-ApiManagementPolicy {
+  <#
+    .SYNOPSIS
+      Confirms that an API Management Policy exists.
+
+    .DESCRIPTION
+      The Confirm-AzBPApiManagementPolicy cmdlet gets an API Management Policy using the specified API, API Management
+      Service, and Resource Group names.
+
+    .PARAMETER ResourceGroupName
+      Specifies the name of the resource group under which an API Management service is deployed.
+
+    .PARAMETER ServiceName
+      Specifies the name of the deployed API Management service.
+
+    .PARAMETER ApiId
+      Specifies the identifier of the existing API. This cmdlet returns the API-scope policy.
+
+    .EXAMPLE
+      Confirm-AzBPApiManagementPolicy -ResourceGroupName "rgbenchpresstest" -ServiceName "servicetest" `
+        -Name "benchpresstest"
```

Comment:
`-Name` -> `ApiId`

Reply:
In this case I chose not to use `Name` as the parameter name, because it is not the "name" of the policy like all of the others, it is the ID of the API that the policy is assigned to.

Reply:
Yup understood, so the example in the comments should be changed to use `-ApiId` instead of `-Name`.

Reply:
Ah, now I get it LOL... sorry, totally misunderstood.

Reply:
Fixed! Thank you!